ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/BasicOptimizeCommit.java

```diff
@@ -228,6 +228,7 @@ private void minorCommit(ArcticTable arcticTable,
       AtomicInteger addedPosDeleteFile = new AtomicInteger(0);
       StructLikeMap<Long> partitionOptimizedSequence =
           TablePropertyUtil.getPartitionOptimizedSequence(arcticTable.asKeyedTable());
+      AtomicBoolean shouldCommit = new AtomicBoolean();
```

Comment:
Maybe we can support `skipEmptyCommit` method in `OverwriteBaseFiles`.

Reply:
It seems to be more reasonable. I will change it.